airflow/configuration.py

```diff
@@ -209,9 +209,9 @@ def _get_cmd_option(self, section, key):
         fallback_key = key + '_cmd'
         # if this is a valid command key...
         if (section, key) in self.as_command_stdout:
-            if super(AirflowConfigParser, self) \
+            if super() \
```

Comment:
This formatting looks a bit odd now, these 2 lines easily fit on a single line. Could you fix these formatting issues?

Reply:
Will start fixing it once https://github.com/apache/airflow/pull/5022 is merged.